QUARKUS-874.md

```diff
@@ -0,0 +1,37 @@
+# QUARKUS-874 Production support for Windows
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-874
+
+The goal is to ensure some level of support and identify existing limitations on Quarkus side for Windows.
+
+## Scope of the testing
+The selected Test Suites will be executed on the selected Windows Platforms to ensure functionality is fully accomplished.
+In order to accommodate this in our current test suites, we need to rename existing jobs in Jenkins in order to differentiate them for those running on RHEL and on Windows platforms. 
+
+### Selected Windows Platforms
+- Windows Server 2019
+
+### Selected Test Suites
+ - Start-Stop Test Suite
+   - In Github CI - for Pull Requests
+   - New Jenkins Jobs
+ - Beefy Test Suite
+   - In Github CI - for Pull Requests and Daily runs
+   - New Jenkins Jobs
+ - QuickStarts Test Suite
+   - New Jenkins Jobs
+ - Combinations of Quarkus extensions Test Suite
+   - In Github CI - for Pull Requests and Daily runs
+ - Workshop Test Suite
+   - New Jenkins Jobs
+
+### Platform testing
+Tests are going to be executed against all tested versions of databases, RHEL and OpenShift versions. 
+Both the JVM and native mode is going to be tested.
+
+## Impact on resources
```

Comment:
Add "Impact on Jenkins" section to mention need of new jobs definitions and implication on some jobs naming unification

Reply:
Done